hbw/app/javascript/packs/hbw/components/shared/hoc/select.js

```diff
@@ -50,6 +51,12 @@ export default WrappedComponent => class WithSelect extends Component {
     if (((a === '') && (b === null)) || ((a === null) && (b === ''))) {
       return true;
     }
+    if (((a === 'null') && (b === null)) || ((a === null) && (b === 'null'))) {
+      return true;
+    }
+    if (multi && (((a?.length === 0) && (b === null)) || ((a === null) && (b?.length === 0)))) {
```

Comment:
Could you provide some case descrption? Can't see why do we need it.

Reply:
In nullable multi = true tables we can deselect all rows so value of the table will be []
    We should parse it like a null value.
    It will be used in hasValueInChoices function